nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -469,10 +471,12 @@ in {
               add_header X-Robots-Tag none;
               add_header X-Download-Options noopen;
               add_header X-Permitted-Cross-Domain-Policies none;
+              add_header Referrer-Policy no-referrer;
               error_page 403 /core/templates/403.php;
               error_page 404 /core/templates/404.php;
               client_max_body_size ${cfg.maxUploadSize};
               fastcgi_buffers 64 4K;
+              fastcgi_hide_header X-Powered-By;
```

Comment:
Good change, should probably be added to the general nginx configuration at some point IMO

Reply:
@dasJ Do you mean global to the nginx module or the global nginx module of nextcloud? Adding it to the global nginx module is probably not a good idea, as there is no way to undo it if somebody wants to not hide the header.

Reply:
@bachp Yes, I mean the global nginx module. It was just a thought, nothing to be added to this PR ;)

Reply:
I did something similar for Apache in https://github.com/NixOS/nixpkgs/commit/1bec75301bd9d15c2ee0be4c587bc652f65df582

I think it's a great idea for another PR 👍